compiler/rustc_const_eval/src/errors.rs

```diff
@@ -271,6 +271,18 @@ pub struct RawBytesNote {
     pub bytes: String,
 }
 
+// FIXME(fee1-dead) do not use stringly typed `ConstContext`
```

Comment:
Did you include this as a reminder to be fixed in this PR or do you want to change this later?

Reply:
I want to change this later, not in this PR